src/components/Header.js

```diff
@@ -1,8 +1,27 @@
 import React from 'react';
 import logo from '../assets/logo.png';
 import { Link } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { auth } from '../firebase/firebase.config';
+import { removeUser } from '../redux/coderSlice';
+import { useDispatch } from 'react-redux';
+import {
+    signOut
+} from 'firebase/auth'
 
 const Header = () => {
+
+    // const userInfo = useSelector((state) => state.codershop.userInfo);
```

Comment:
Why is this code commented out? Saving it here for future reference? If not, let's go ahead and delete it.